### PR TITLE
Move help files to correct (more logical) location

### PR DESCRIPTION
**Pull request by renea**
Move help files to correct location, partial for [JENKINS-29079](https://issues.jenkins-ci.org/browse/JENKINS-29079)

### CODE REVIEW

File: src/main/resources/hudson/scm/SubversionSCM/ModuleLocation/help-local_ja.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<div>
-  このモジュールをチェックアウトするローカルのディレクトリ(<a href="ws" target="_blank">ワークスペースのルート</a>からの相対パス)
-  を指定してください。
-  何も設定しないと、コマンドの<tt>svn</tt>と同様に、URLの最後のパス名をデフォルトとして使用します。
-</div>
+<div>
```
Comment (oleg-nenashev) [inline src/main/resources/hudson/scm/SubversionSCM/ModuleLocation/help-local_ja.html]: Seems something has been changed here (encoding, etc.)

**Comment by oleg-nenashev** [comment]
Looks good to me (exception Japanese localizations)

**Comment by recena** [comment]
@renea What is the relation with [JENKINS-29079](https://issues.jenkins-ci.org/browse/JENKINS-29079)?

File: src/main/resources/hudson/scm/SubversionSCM/ModuleLocation/help-local_ja.html (hunk 1 above)
Comment (renea) [inline src/main/resources/hudson/scm/SubversionSCM/ModuleLocation/help-local_ja.html]: Git converted line endings from windows style to unix style. 
All other files appear to use unix style line endings.

**Comment by renea** [comment]
@recena,  No help was shown for the fields. 
Showing the correct help texts will reduce the unclarity a lot.

**Comment by recena** [comment]
@renea I'll try your PR later. If everything is fine, we can merge it

**Comment by recena** [comment]
@renea The help texts are shown anyway. This change is only for re-structure the source code.

**Comment by recena** [comment]
@christ66 Do you agree with these changes?

**Comment by renea** [comment]
@recena , strange, for me in 2.5.0 (?) I did not see the help. (no access to the server now to check).
Well, as long as it is fixed, I don't care if it was by my pr or someone else's change.

**Comment by recena** [comment]
@renea Probably, my mistake.

**Comment by recena** [comment]
@renea We are going to include your PR in the next release. Thanks so much for your colaboration.

**Pull request merged**